pkgs/data/misc/iana-etc/default.nix

```diff
@@ -1,17 +1,18 @@
-{stdenv, fetchurl}:
+{ stdenv, cacert, curl }:
 
-stdenv.mkDerivation rec {
-  name = "iana-etc-2.30";
-
-  src = fetchurl {
-    url = "http://sethwklein.net/${name}.tar.bz2";
-    sha256 = "03gjlg5zlwsdk6qyw3v85l129rna5bpm4m7pzrp864h0n97qg9mr";
-  };
 
-  preInstall = "installFlags=\"PREFIX=$out\"";
+stdenv.mkDerivation rec {
 
+  name = "iana-etc-${version}";
+  version = "20170321";
+  
+  phases = [ "buildPhase" ];
+  buildInputs = [ cacert curl ];
+  builder = ./builder.sh;
```

Comment:
You can only access the network with a fixed output drv when sandboxing is enabled, I'm pretty sure this will fail on hydra.